src/core/mount-setup.c

```diff
@@ -337,6 +339,31 @@ static int nftw_cb(
 
         return FTW_CONTINUE;
 };
+
+static int relabel_cgroup_filesystems(void) {
+        int r;
+        struct statfs st;
+
+        /* Temporarily remount the root cgroup filesystem to give it a proper label. Do this
+           only when the filesystem has been already populated by a previous instance of systemd
+           running from initrd. Otherwise don't remount anything and leave the filesystem read-write
+           for the cgroup filesystems to be mounted inside. */
+        r = statfs("/sys/fs/cgroup", &st);
+        if (r < 0) {
```

Comment:
We don't use curly braces for simple ifs, but that's just a cosmetic "issue". Also, the same has been committed upstream.